### PR TITLE
Source tool

### PR DESCRIPTION
**Pull request by PaulakaPaul**
I would like a review over my code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,77 +1,77 @@
...
-venv/
-ENV/
-env.bak/
-venv.bak/
-
-*.sqlite3
-
-# IDE
-.idea
+# Created by .ignore support plugin (hsz.mobi)
```
Comment (lucifurtun) [inline .gitignore]: @PaulakaPaul, please do not forget to undo this permission changes.

Comment (PaulakaPaul) [reply .gitignore]: Ok, I will undo it.

Comment (PaulakaPaul) [reply .gitignore]: I don't want so sound stupid, but I will have to. What permission changes ? I looked over my commit, but I don't get it.

Comment (lucifurtun) [reply .gitignore]: @PaulakaPaul, isn't it weird that your PR looks like you made some changes (https://github.com/safefleet/labs-server/pull/10/files) in files like `.gitignore` but actually you didn't make any change ? You probably changed some file permissions.

**Comment by lucifurtun** [comment]
You also need to fix the conflicts. Otherwise, we can't merge this PR.

**Comment by PaulakaPaul** [comment]
I resolved the conflicts. Hope it's ok now.

**Pull request merged**